.gitignore

```diff
@@ -14,6 +14,8 @@ _site
 # Compiled Files
 /assets/css/*.css
```

Comment:
I believe you can make this `/assets/css/**/*.css` and that will recursively ignore any `.css` file, regardless of subdirectory. Might be worth testing instead of separately declaring `/assets/css/interactive`. Same for the JS files. Although if this doesn't work, then what you had is great.